solutions/Scientific-tree-biomass-challenge-R.txt

```diff
@@ -1 +1 @@
-[1] 151992268
\ No newline at end of file
+[1] 26710042
```

Comment:
This is the output solution for http://www.datacarpentry.org/semester-biology/exercises/Scientific-tree-biomass-challenge-R/. I found the solution to be wrong during the revision of this exercise to `Tidyr-tree-biomass-R.R`. This exercise only asks for the single value of total forest biomass.